include/votca/tools/akimaspline.h

```diff
@@ -15,6 +15,7 @@
  *
  */
 
+#pragma once
```

Comment:
why add this?

Reply:
"In the C and C++ programming languages, #pragma once is a non-standard but widely supported preprocessor directive designed to cause the current source file to be included only once in a single compilation. Thus, #pragma once serves the same purpose as include guards, but with several advantages, including: less code, avoidance of name clashes, and sometimes improvement in compilation speed."